_aggregations/pipeline/bucket-sort.md

```diff
@@ -0,0 +1,229 @@
+| `from`           | Optional          | String          | The index of the first result to return. Must be a non-negative integer. Default is `0`. See [The `from` and `size` parameters]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/paginate/#the-from-and-size-parameters). |
+| `size`           | Optional          | String          | The maximum number of results to return. Must be a positive integer. See [The `from` and `size` parameters]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/paginate/#the-from-and-size-parameters).|
+
+You must supply at least one of `sort`, `from`, and `size`.
+{: .note}
+
+## Example
+
+The following example creates a date histogram with a one-month interval from the OpenSearch Dashboards e-commerce sample data. The `sum` subaggregation calculates the sum of all bytes for each month. Finally, the aggregation sorts the buckets in descending order of number of bytes:
+
+```json
+GET opensearch_dashboards_sample_data_logs/_search
+{
+  "size": 0,
+  "aggs": {
+    "sales_per_month": {
+      "date_histogram": {
+        "field": "@timestamp",
+        "calendar_interval": "month"
+      },
+      "aggs": {
+        "total_bytes": {
+          "sum": {
+            "field": "bytes"
+          }
+        },
+        "bytes_bucket_sort": {
+          "bucket_sort": {
+            "sort": [
+              { "total_bytes": { "order": "desc" } }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Example response
+
+The aggregation reorders the buckets descending order of total bytes:
+
+```json
+{
+  "took": 3,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 10000,
+      "relation": "gte"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "sales_per_month": {
+      "buckets": [
+        {
+          "key_as_string": "2025-05-01T00:00:00.000Z",
+          "key": 1746057600000,
+          "doc_count": 7072,
+          "total_bytes": {
+            "value": 40124337
+          }
+        },
+        {
+          "key_as_string": "2025-06-01T00:00:00.000Z",
+          "key": 1748736000000,
+          "doc_count": 6056,
+          "total_bytes": {
+            "value": 34123131
+          }
+        },
+        {
+          "key_as_string": "2025-04-01T00:00:00.000Z",
+          "key": 1743465600000,
+          "doc_count": 946,
+          "total_bytes": {
+            "value": 5478221
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+## Example: Truncating the results 
+
+To truncate the results, provide the `from` and/or `size` parameters. The following example performs the same sort, but returns two buckets, starting with the second bucket:
```

Comment:
2nd sentence: Delete the comma after "sort". I would also add "operation" after "sort".